block_iagora.php

```diff
@@ -22,13 +22,14 @@
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
+  
 /**
- * The iagora block class.
+ * Stores configuration data for the block instance.
```

Comment:
```suggestion
 * Store configuration data for the iagora block.
```
As most docstrings are now in imperative style, let's use the imperative style consistently)